ideasbox/templates/ideasbox/pagination.html

```diff
@@ -2,13 +2,24 @@
 {% if is_paginated %}
     <div class="pagination">
         {% if page_obj.has_previous %}
-            <a href="{{ base_url }}?page={{ page_obj.previous_page_number }}" class="previous">{% trans "previous" %}</a>
+
+            {% if page_obj.previous_page_number != 1 %}
```

Comment:
this seems to break some tests. Maybe this is the test to be fixed, as the change looks OK.